.config/nextest.toml

```diff
@@ -0,0 +1,5 @@
+[profile.default]
+default-filter = "not test(/test_k8s_integration_/)"
+
+[profile.k8s_integration]
+default-filter = "test(/test_k8s_integration_/)"
```

Comment:
open to other prefix suggestions

Reply:
is the only way to do this prefixes? it just feels a bit "magic"

Reply:
we can instead separate tests that require tilt into a `tests/` directory and segment based on test location
I weakly prefer to keep tests colocated

Reply:
agreed, thats fine then. Can we just comment the tests using this pattern with that explicitly for now.